content/ko/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md

```diff
@@ -163,8 +169,9 @@ pods "constraints-cpu-demo-2" is forbidden: maximum cpu usage per Container is 8
 kubectl apply -f https://k8s.io/examples/admin/resource/cpu-constraints-pod-3.yaml --namespace=constraints-cpu-example
 ```
 
-컨테이너가 너무 작은 CPU 요청량을 지정하므로, 출력 결과에 파드가 생성되지
-않은 것으로 표시된다.
+결과를 보면 파드가 생성되지 않은 것을 확인할 수 있으며, 이는 해당 파드가 수용 불가능한 컨테이너를 정의하고 있기 때문이다.
+해당 컨테이너가 수용 불가능한 이유는 
+지정된 최저 CPU 상한보다도 낮은 CPU 상한을 지정하고 있기 때문이다.
```

Comment:
섹션 제목이 "최소 CPU **요청량**을 충족하지 않는 파드 생성 시도"
(원문: "Attempt to create a Pod that does not meet the minimum CPU **request**") 인데

이 부분에서는 **상한(limit)** 에 대해 이야기하고 있습니다.
원문:
> The output shows that the Pod does not get created, because it defines an unacceptable container.
That container is not acceptable because it specifies a CPU **limit** that is lower than the
enforced minimum:

아마 원문이 잘못되어 있는 것 같습니다. 😅
(cf. `memory-constraint-namespace.md` 에는 제대로 되어 있습니다.)

원문 수정을 제안하는 이슈를 올렸습니다. (#31856) 😊